2022q4/releng.adoc

```diff
@@ -0,0 +1,23 @@
+=== FreeBSD Release Engineering Team
+
+Links: +
+link:https://www.freebsd.org/releases/12.4R/schedule/[FreeBSD 12.4-RELEASE schedule] URL: link:https://www.freebsd.org/releases/12.4R/schedule/[https://www.freebsd.org/releases/12.4R/schedule/] +
+link:https://www.freebsd.org/releases/12.4R/announce/[FreeBSD 12.4-RELEASE announcement] URL: link:https://www.freebsd.org/releases/12.4R/announce/[https://www.freebsd.org/releases/12.4R/announce/] +
+link:https://www.freebsd.org/releases/13.2R/schedule/[FreeBSD 13.2-RELEASE schedule] URL: link:https://www.freebsd.org/releases/13.2R/schedule/[https://www.freebsd.org/releases/13.2R/schedule/] +
+link:https://www.freebsd.org/releases/14.0R/schedule/[FreeBSD 14.0-RELEASE schedule] URL: link:https://www.freebsd.org/releases/14.0R/schedule/[https://www.freebsd.org/releases/14.0R/schedule/] +
+link:https://download.freebsd.org/snapshots/ISO-IMAGES/[FreeBSD development snapshots] URL: link:https://download.freebsd.org/snapshots/ISO-IMAGES/[https://download.freebsd.org/snapshots/ISO-IMAGES/]
+
+Contact: FreeBSD Release Engineering Team, <re@FreeBSD.org>
+
+The FreeBSD Release Engineering Team is responsible for setting and publishing release schedules for official project releases of FreeBSD, announcing code freezes and maintaining the respective branches, among other things.
+
+During the fourth quarter of 2022, the Release Engineering Team completed work on the 12.4-RELEASE cycle.
+This is the final release from the stable/12 branch.
+During the release cycle, only one BETA builds and two RC (release candidate) builds were needed; overall the release cycle went very smoothly and the release took place on December 5th.
+
+During the fourth quarter of 2022, the Release Engineering Team continued providing weekly development snapshot builds for the *main*, *stable/13*, and *stable/12* branches.

```

Comment:
> During the fourth quarter of 2022, the Release Engineering Team

The previous paragraph began with the same phrase, and it's all in the context of a quarterly report for the specified period. 

Instead: 

> The Team